### ISSUE TITLE
Not working with latest GeForce Experience 2.11.3.5

### ISSUE THREAD

**Issue by madwedge** [moonlight-stream/moonlight-ios#224]
Updated GE to latest 2.11.3.5 yesterday and now connection fails after Handshake. Have tried this on iOS using my iPhone and on Android using nVidia Shield TV. Both fail.

**Comment by cgutman** [comment]
2.11.3.5 is fully supported with the current Moonlight releases on all platforms. Can you better describe the failure you're seeing?

**Comment by madwedge** [comment]
Thanks for the response. I'm on my home WiFi so on same network. I go to launch a game, or Steam Big Picture Mode, and is says "Starting app", and then "RTSP Handshake" but then says "Connection Terminated" and goes back to the list of games, however on my PC, Steam Big Picture has actually launched. Same as on Android nVidia Shield

**Comment by cgutman** [comment]
Can you confirm that Shield Hub streams correctly?

**Comment by madwedge** [comment]
I tried that as well. The Sheild Hub just kept asking me to update GeForce Experience before I could use it. I may have a Beta version of Shield Hub though compared to a non Beta GE on my PC. I've always had issues with Shield Hub though, that's why I prefer to use Moonlight.

**Comment by cgutman** [comment]
OK, can you upload a zip of the C:\ProgramData\NVIDIA Corporation\NvStream folder?

ProgramData is a hidden folder so you may need to show hidden files to see it.

**Comment by madwedge** [comment]
I've managed to solve it. The other day I installed KinoConsole Server on my PC. I assume you know it's another game streaming service. I wanted to try it out. It seemed to have interfered with Moonlight in some fashion. I've just uninstalled KinoConsole Server from my PC, tried Moonlight again and it's working fine now. Very strange, however good news. 

I had also tried completely reinstalling GeForce Experience on my PC and the Moonlight app on my iPhone but this didn't help. Definitely seemed to be this KinoConsole. Sorry to have wasted your time. Keep up the great work. :-)
